quinterojose/KeePassDPG
# Updated project to .NET Framework 4.8.

ChangleLog:
    * modify: bump version to 1.2.0 in src/KeePassDPG/Properties/AssemblyInfo.cs
    * create PLGX from updated local source files
    * test new PLGX version
    * create sha256 hashes of PLGX files for original author's review
    * store review files in <project-root>/PLGX/ directory
    * take screenshots of errant behavior and fixed behavior
    * store screenshots in PLGX/ directory beside their respective files

Signed-off-by: William Smyth <codedchaos@zohomail.com>

Commit History:

 > This is the commit message #2:
```
    Create test/proof files
   
    - Add PLGX file newly generated with fix
    - Add original PLGX file from author's releases
    - create checksums of PLGX using sha256
    - test sha256 hashes for PLGX files against shasum.bat to ensure
      they match the same hashes produced in BASH
   
    Signed-off-by: William Smyth <codedchaos@zohomail.com>
```

 > This is the commit message #3:
```
    Add screenshots of KeePass UI -> Plugins
   
    - show original PLGX from release by original author
    - show updated PLGX after modification of AssemblyInfo.cs
   
    Signed-off-by: William Smyth <codedchaos@zohomail.com>
```
